web/client/epics/identify.js

```diff
@@ -174,7 +174,7 @@ module.exports = {
             const {disableAlwaysOn = false} = (store.getState()).mapInfo;
             return disableAlwaysOn || !stopFeatureInfo(store.getState() || {});
         })
-            .map(({point, layer}) => featureInfoClick(point, layer)),
+            .switchMap(({point, layer}) => Rx.Observable.of(featureInfoClick(point, layer), mapPopupUpdate('identify', { coordinates: point ? point.rawPos : [] }))),
```

Comment:
lat-lon should be enough... anyway, for the moment this part is not needed. revert all the parts concerning identify